backend/src/test/java/de/nordakademie/iaa/noodle/dao/SurveyRepositoryTest.java

```diff
@@ -0,0 +1,12 @@
+package de.nordakademie.iaa.noodle.dao;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+class SurveyRepositoryTest {
+
+    @Autowired
+    private SurveyRepository surveyRepository;
+
+}
```

Comment:
Does it make sense to add an empty test file?

Reply:
Guess not. Deleted until I get around to properly work on the actual DAOs + Tests.